lib/usd/hdMaya/adapters/aiSkydomeLightAdapter.cpp

```diff
@@ -65,7 +65,11 @@ class HdMayaAiSkyDomeLightAdapter : public HdMayaLightAdapter
             return VtValue(light.findPlug("aiExposure", true).asFloat());
         } else if (paramName == HdLightTokens->normalize) {
             return VtValue(light.findPlug("aiNormalize", true).asBool());
+#if USD_VERSION_NUM >= 2102
```

Comment:
For my own education, when should we use `USD_VERSION_NUM` and when should we use `HD_API_VERSION`?

Reply:
Yeah, sorry, this is admittedly not very obvious.

Internally, our development cadence runs faster than the public USD release cycle, and the Hydra API in particular is changing fairly rapidly. We've been needing more fine-grained API guards to deal with that as we attempt to support several releases and mixed-component builds (internal-only stuff built on top of USD) across multiple shows. There isn't really a rigid structure around when API-level guards (`HD_API_VERSION`, `USD_IMAGING_API_VERSION`, etc.) get added or incremented. It's just done as needed to address a particular API change.

So externally to Pixar, the go-to should generally be `USD_VERSION_NUM`, since we don't really intend to support individual components of the public USD distribution being built separately from each other.